examples/cis2-wccd/Cargo.toml

```diff
@@ -6,7 +6,8 @@ edition = "2018"
 license = "MPL-2.0"
 
 [features]
-default = ["std"]
+default = ["std", "crypto-primitives"]
```

Comment:
Do we need crypto-primitives in the default? I thought we did not use that in wccd?

Reply:
We use it in the test cases now but not in the smart contract itself that is why I included it here.